pages/Developer/MAP API/map-migration-api.md

```diff
@@ -0,0 +1,164 @@
+- `ec2_commercial_db_entity`: Specific incentives related to EC2 commercial database.
+- `sap_entity`: Specific incentives related to SAP.
+- `baseline_s3_entity`: Specific incentives related to S3.
+- `speciality_services_entity`: Specific incentives related to speciality services.
+- `private_equity_entity`: Specific incentives related toprivate equity.
+- `tagged_resources`: Information about tagged resources, including resource count and cost.
+- `untagged_resources`: Information about untagged resources, including resource count and cost.
+- `credits`: Information about received credits, including records of credit details.
+- `created`: The creation date of the project.
+- `modified`: The last update date of the project.
+- `name`: The name of the project.
+- `migration_id`: The migration ID is identified by the project number in the contract, of the form MPExxxxx (where xxxxx are digits) or a 10-digit identifier such as ABCLMNOPYZ. If you are not sure where to find your MAP Project ID, refer to the top of the first page of your MAP contract.
+- `tag_key`: Must be the exact value string from your AWS contract. This value should be mig followed by the 5 or 10-digit MAP ID. If it’s an older MAP contract, you may also use the server-id (typically d- followed by 9 digits).
+- `server_id`: The server ID, as referred above.
+- `credit_percent`: Prefilled for you to the typical 25%, however you may edit this when creating a project in nOps if your contract terms state a different discount for standard MAP resources.
+- `workload_pk`: Primary key of a workload (deprecated).
+- `date_start`: The start date of the migration (per your MAP contract terms from AWS).
+- `date_end`: The end date of the migration (per your MAP contract terms from AWS).
+- `migration_period`: The migration period in months.
+- `windows_optimization`: Indicates if Windows optimization is enabled.
+- `map_credit_cap`: The MAP credit cap (default is 1.000.000USD).
+- `client`: The ID of the client associated with the project.
+
+---
+
+### 2. Retrieve a Specific Map Migration Project
+
+- **URL**: `/api/map-migration/{id}/`
+- **Method**: `GET`
+- **Description**: Retrieves details of a specific map migration project.
+
+#### Request Parameters
+
+- **Path Parameter**:
+  - `id` (required): The ID of the map migration project.
+
+#### Response Format
+
+The response is a JSON object like the one above
+
+### 3. List Products in a Map Migration Project
+
+- **URL**: `/api/map-migration/{id}/products/`
+- **Method**: `GET`
+- **Description**: Lists all the products in a map migration project.
+
+#### Request Parameters
+
+- **Path Parameter**:
+  - `id` (required): The ID of the map migration project.
+
+#### Response Format
+
+The response is a JSON array containing a list of products in the map migration project. Each product has the following fields:
+
+- `product_product_name`: The name of the product.
```

Comment:
`product_product_name`: The name of the product (a product here refers to an AWS Service, example: EC2, RDS etc.)